packages/slice-machine/src/features/customTypes/customTypesBuilder/SliceZoneBlankSlate.tsx

```diff
@@ -34,58 +35,62 @@ export const SliceZoneBlankSlate: FC<SliceZoneBlankSlateProps> = ({
     <Box
       flexGrow={1}
       justifyContent="center"
+      alignItems="center"
+      padding={32}
       data-testid="slice-zone-blank-slate"
     >
-      <BlankSlate backgroundImage="/blank-slate-slice-zone.png">
-        <BlankSlateContent>
-          <Box justifyContent="center" padding={{ bottom: 16 }}>
-            <SliceMachinePrinterIcon />
-          </Box>
-          <BlankSlateTitle>Add slices</BlankSlateTitle>
-          <BlankSlateDescription>
-            Slices are website sections that you can reuse on different pages
-            with different content. Each slice has its own component in your
-            code.
-          </BlankSlateDescription>
-          <BlankSlateActions>
-            <ActionList>
-              {aiSliceGenerationExperiment.eligible && (
-                <ActionListItem
-                  startIcon="autoFixHigh"
-                  onClick={openGenerateSliceWithAiModal}
-                  description="Let AI instantly create a Slice for you."
-                >
-                  Generate with AI
-                </ActionListItem>
-              )}
+      <BlankSlate>
+        <Box justifyContent="center" padding={{ bottom: 16 }}>
+          <BlankSlateIcon
+            lineColor="purple9"
+            backgroundColor="purple5"
+            name="add"
+            size="large"
+          />
+        </Box>
+        <BlankSlateTitle size="big">Add slices</BlankSlateTitle>
+        <BlankSlateDescription>
+          Slices are website sections that you can reuse on different pages with
+          different content. Each slice has its own component in your code.
+        </BlankSlateDescription>
+        <BlankSlateActions>
+          <ActionList>
+            {aiSliceGenerationExperiment.eligible && (
+              <ActionListItem
+                startIcon="autoFixHigh"
+                onClick={openGenerateSliceWithAiModal}
+                description="Let AI instantly create a Slice for you."
+              >
+                Generate with AI
+              </ActionListItem>
+            )}
+            <ActionListItem
+              startIcon="add"
+              onClick={openCreateSliceModal}
+              description="Build a custom Slice your way."
+            >
+              Start from scratch
+            </ActionListItem>
+            {isSlicesTemplatesSupported ? (
+              <ActionListItem
+                startIcon="contentCopy"
+                onClick={openSlicesTemplatesModal}
+                description="Choose from ready-made examples."
+              >
+                Use a template
+              </ActionListItem>
+            ) : undefined}
```

Comment:
same as above, you can avoid the undefined, it happens in other places in this pr or around the code that we have touched, but it's not important
```suggestion
            {isSlicesTemplatesSupported && (
              <ActionListItem
                startIcon="contentCopy"
                onClick={openSlicesTemplatesModal}
                description="Choose from ready-made examples."
              >
                Use a template
              </ActionListItem>
            )}
```

Reply:
Good call. Done ✔️